tests/docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
       - cluster.name=docker-test-cluster
       - node.name=docker-test-node-1
       # Test setting max heap to a non default value with environment variables.
-      - "ES_JAVA_OPTS=-Xms1g -Xmx1g"
+      - "ES_JAVA_OPTS=-Xms1151M -Xmx1151M"
```

Comment:
Deliberately used an unusual number here. It's less likely to be masked by a default in the future.